core/src/main/resources/swagger/api-docs.yaml

```diff
@@ -5625,7 +5625,27 @@ components:
             Sam.
         managedAppCoordinates:
           $ref: '#/components/schemas/AzureManagedAppCoordinates'
+        members:
+          description: The list of members to add to the billing project
+          type: array
+          items:
+            $ref: '#/components/schemas/ProjectAccessUpdate'
+        inviteUsersNotFound:
+          description: If true, invite the users to Terra if they are not registered in the system. They will be given access to the project
```

Comment:
I was at first reading this as whether or not to email an invitation. But it's actually whether or not you are allowed to specify users who are not already registered in Terra, correct?